runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/OpenSourceSection/OpenSourceBackground/OpenSourceBackground.tsx

```diff
@@ -0,0 +1,15 @@
+import React, { FC } from 'react';
+
+import { OPEN_SOURCE_SECTION_ID } from '#src-landing/utils/utils';
+
+import styles from './OpenSourceBackground.module.scss';
+
+const OpenSourceBackground: FC = ({ children }) => (
```

Comment:
isnt there any TS error? im sure that FC from react 18 does not contain `children` by default

Reply:
no, there isn't